.github/workflows/native.yml

```diff
@@ -47,3 +47,43 @@ jobs:
         working-directory: ./native
       - run: npm run test
         working-directory: ./native
+
+  lint:
+    name: Lint
+    needs: changes
+    if: ${{ needs.changes.outputs.native == 'true' }}
```

Comment:
This is neat! Haven't seen the `dorny/paths-filter` action before. Does it provide more advanced features beyond the standard [`paths` config](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)?

Reply:
Honestly, unsure, based this on `frontend.yml` which used it as well.